src/Compilers/CSharp/Test/Emit3/Semantics/CollectionExpressionTests_WithElement_Extra.cs

```diff
@@ -506,17 +504,37 @@ static ICollection<T> Create<T>(params T[] items)
         string expectedIL;
         switch (argsPrefix)
         {
+            case "with(), ":
```

Comment:
thse test changes are desirable.  previously, despite having `with()` we were still bypassing call'ing list's no-arg constructor, and were instead still explicitly generating the list with teh values we thought it should have.  that's not appropriate.  If the user supplies a direct request to call a particular constructor, we want to honor it, as this is their escape hatch for not liking us determining the capacity ourselves.